bootstrap/kubeadm/main.go

```diff
@@ -123,6 +125,12 @@ func InitFlags(fs *pflag.FlagSet) {
 	fs.DurationVar(&syncPeriod, "sync-period", 10*time.Minute,
 		"The minimum interval at which watched resources are reconciled (e.g. 15m)")
 
+	fs.Float32Var(&restConfigQPS, "kubernetes-client-qps", 20,
+		"Maximum queries per second that should be allowed from the controller to the API server")
```

Comment:
Copying from the Kubernetes docs:

```suggestion
		"Maximum queries per second from the controller client to the Kubernetes API server. Defaults to 20")
```

Reply:
Fixed!